h2o-algos/src/main/java/hex/coxph/CoxPHModel.java

```diff
@@ -51,6 +51,8 @@
 
     public boolean _calc_cumhaz = true; // support survfit
 
+    public boolean _single_node_mode = false;
```

Comment:
seem unused

Reply:
it is used all over the place :)

Reply:
on sorry, i thought this was declared on model class, I was only looking the file name and did not realize this is on params